Utilities/JIT.h

```diff
@@ -135,15 +136,12 @@ class jit_compiler final
 	// Local LLVM context
 	llvm::LLVMContext m_context;
 
-	// JIT Event Listener
-	std::unique_ptr<struct EventListener> m_jit_el;
```

Comment:
p

Reply:
can fix the problems and marigle into master build plaese

Reply:
@mew21dot this is not a place for you to be a whiny arse, now stop spamming the tracker of things you don't understand.